packages/manager/src/queries/placementGroups.ts

```diff
@@ -27,13 +29,25 @@ import type {
   UpdatePlacementGroupPayload,
 } from '@linode/api-v4';
 
-export const queryKey = 'placement-groups';
+export const placementGroupQueries = createQueryKeys('placement-groups', {
+  all: (params: Params = {}, filters: Filter = {}) => ({
+    queryFn: () => getAllPlacementGroupsRequest(),
+    queryKey: [params, filters],
+  }),
```

Comment:
Looks like `params` and `filters`  are unused here. If they are not needed, we can remove them

```suggestion
  all: {
    queryFn: getAllPlacementGroupsRequest,
    queryKey: null,
  },
```

Reply:
ah yes, much cleaner 👍 - done